cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/dataset/Metadata.java

```diff
@@ -27,19 +27,19 @@
 /**
  * Represents the complete metadata of a {@link Id.NamespacedId} including its properties and tags.
  */
-public class MetadataHistoryEntry {
+public class Metadata {
```

Comment:
There is already a MetadataEntry class which we might remove later. So far now we can't rename it to that sadly.